package.json

```diff
@@ -17,6 +17,7 @@
   ],
   "devDependencies": {
     "@architect/architect": "^6.3.5",
+    "cheerio": "1.0.0-rc.3",
```

Comment:
Does cheerio go here, or elsewhere, @ryanblock ?

Reply:
We had some chats about this and this seems legit (dev dep shouldn't end up in prod).